packages/riipen-ui-docs/src/pages/components/menu/Simple.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import Menu from "@riipen-ui/components/Menu";
+import MenuItem from "@riipen-ui/components/MenuItem";
+import Button from "@riipen-ui/components/Button";
+
+export default function Simple() {
```

Comment:
Can we use function components now? I know for a while we said class only everywhere.. But I do love function components and think they are the future of React :shrug:

Reply:
We had to use some function components for a few very specific menu things. I would say when they are warranted use them.

I still personally don't like hooks :(.

Reply:
I made a comment in the dev channel to get everyone's thoughts, feel free to post your dislike there :p